packages/wonder-blocks-dropdown/src/hooks/use-listbox.tsx

```diff
@@ -70,7 +76,7 @@ export function useListbox({
     };
 
     const focusPreviousItem = React.useCallback(() => {
-        if (focusedIndex === 0) {
+        if (focusedIndex <= 0) {
```

Comment:
**note:** Found an issue when the focus was on the input and I pressed ArrowUp. The problem was that the listbox visual focus was lost in that case, but this is fixed with that change.


Uploading Screen Recording 2024-08-29 at 5.54.08 PM.mov…

Reply:
oh oops, here it is for posteriority 😄 

https://github.com/user-attachments/assets/301abd3d-f012-42f2-a8ea-19eda1d99c80